.npmrc

```diff
@@ -2,4 +2,5 @@
 # packages as it casues issues with manypkg
 # @link https://pnpm.io/npmrc#prefer-workspace-packages
 save-workspace-protocol=false
-prefer-workspace-packages=true
\ No newline at end of file
+prefer-workspace-packages=true
+shamefully-hoist=true
```

Comment:
i prefer public-hoist-patterns so we're not hoisting more than we need to. See https://github.com/t3-oss/t3-env/blob/main/.npmrc